projects/ci-cd.md

```diff
@@ -0,0 +1,143 @@
+| pytest         | [pyTest OpenTelemetry plugin](https://pypi.org/project/pytest-otel/)                                                                                        |                                                                                                    | * Tracing of Python test                                      |
+| Maven          | [Maven OpenTelemetry extension](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/maven-extension/README.md)                           |                                                                                                    | * Tracing of Maven builds                                     |
+| Ansible        | [Ansible OpenTelemetry Callback Plugin](https://docs.ansible.com/ansible/latest/collections/community/general/opentelemetry_callback.html)                  |                                                                                                    | * Tracing in Ansible playbooks                                |
+| JUnit          | [JUnit to OpenTelemetry](https://github.com/mdelapenya/junit2otlp)                                                                                          |                                                                                                    | * Tracing of JUnit test results                               |
+| Eiffel         | [Eiffel CI/CD Event Protocol](https://eiffel-community.github.io/)                                                                                          |                                                                                                    | * Event protocol for CI/CD observability and interoperability |
+| CDEvents       | [CD Event Protocol](https://cdevents.dev/)                                                                                                                  |                                                                                                    | * Event protocol for CI/CD observability and interoperability |
+| Tekton         | [Tracing Tekton Enhancement Proposal](https://github.com/tektoncd/community/blob/main/teps/0124-distributed-tracing-for-tasks-and-pipelines.md)             |                                                                                                    | * Tekton Task and Pipeline Tracing                            |
+
+## Staffing / Help Wanted
+
+### TC Sponsorship
+
+Initial TC Contact: @carlosalberto 
+
+> Decision was made in the TC syncs that: 
+> No fulltime TC sponsorship needed at this time because there are no immediate major
+> Specification changes are foreseen. 
```

Comment:
it's not clear to me what this means in practice, do we need this language, or is it ok to just list sponsors similar to other projects?

Reply:
@trask , I was conversing with @carlosalberto about this the day after the decision was made to go ahead with the SIG where Carlos was initial TC contact, and was asked to include a note that capture this as the current understanding.

However, based on the maintainer meeting that followed, where the "no full time TC sponsorship needed" conversation occurred after this approval and due to @jsuereth also being willing to be added as a TC sponsor in the above comment, I'm totally fine with removing this verbiage if, between you three, agree! 😊

Reply:
great! I pinged @carlosalberto to see if he is willing/able to be a TC sponsor instead of just an initial TC contact

Reply:
I confirm I will be a TC sponsor.

Reply:
@adrielp I think we're good to remove the extra verbiage, thx!

Reply:
Done ✅